src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -243,6 +243,26 @@ CHIP_ERROR GenericConfigurationManagerImpl<ImplClass>::StoreManufacturingDate(co
 template <class ImplClass>
 void GenericConfigurationManagerImpl<ImplClass>::InitiateFactoryReset()
 {
+#if CHIP_ENABLE_ROTATING_DEVICE_ID
+    // Increment life time counter to protect against long-term tracking of rotating device ID.
+    _IncrementLifetimeCounter();
+#endif
+    // Inheriting classes should call this method so the lifetime counter is updated if necessary.
+}
+
+template <class ImplClass>
+void GenericConfigurationManagerImpl<ImplClass>::StartAdvertisement()
```

Comment:
Why is this adding two copies of StartAdvertisement?

Reply:
sorry, that was a merge conflict issue, will fix in my next commit